src/main/java/com/noticemedan/mappr/model/pathfinding/Dijkstra.java

```diff
@@ -38,19 +40,14 @@ public Dijkstra(Network n, PathNode s, TravelType travelType) {
 		priorityQueue.insert(s.getId(), distTo[s.getId()]);
 		while (!priorityQueue.isEmpty()) {
 			int vId = priorityQueue.delMin();
-			n.getAllNodes().get(vId).getEdges().forEach(this::relax);
+			n.getAllNodes().get(vId).getEdges().filter(this::hasTravelType).forEach(this::relax);
```

Comment:
Begrænser det her ikke til at den kun finder ruter hvor man kan kører hele vejen med den valgte TravelType?? er vi ikke mere interesseret i at den finder ruten, og derefter tjekker hvor langt man komme på den rute med den valgte TravelType? Hvis jeg f.eks. sætter et slutpunkt midt ude i en skov, så skal den vel stadig finde en rute også selvom jeg har valgt TravelType.CAR.. så skal den bare tage mig så tæt på skoven som muligt?

Reply:
Er ikke helt med på hvad du mener?
Hvis du vælger car, skal den udelukkende finde veje hvor det er tilladt at køre i bil.
Mange af edgene har både "car" og "bike" fx, de bliver selvfølgelig også talt med.

Så hvis du spørger om den her finder veje som udelukkende har "car" som allowed, så er svaret nej, det gør den ikke :)

Reply:
Ahh, forstår hvad du mener nu.
Nej, hvis ikke du kan komme hele vejen i fx bil, så får du en "path cannot be found".
Hvis den så skal vise hvor langt du rent faktisk kan komme, så bliver det hele mere kompliceret. 
Det er vores system ikke som sådan gearet til lige nu.

Tænker den skal vise @EliasJorgensen fine popup hvor der står "kan ikke finde en rute, prøv en anden transport type" eller noget lignende.